npm/webpack-dev-server/src/aut-runner.ts

```diff
@@ -22,6 +22,7 @@ export function init (importPromises, parent = (window.opener || window.parent))
   }
 
   Cypress.onSpecWindow(window, importPromises)
+  Cypress.action('app:window:before:load', window)
```

Comment:
What does this do?

Reply:
This enables usage [window:before:load event](https://docs.cypress.io/api/events/catalog-of-events.html#Window-Before-Load) both for us and for users